samples/snippets/fsharp/VS_Snippets_CLR_System/system.boolean.structure/fs/tostring2.fs

```diff
@@ -0,0 +1,13 @@
+module tostring2
+
+// <Snippet4>
+let raining = false
+let busLate = true
+
+printfn "It is raining: %s" (if raining then "Yes" else "No")
```

Comment:
I think we're preferring interpolation generally here, so these could be 
```suggestion
printfn $"It is raining: %s{if raining then "Yes" else "No"}" 
```

instead if you wanted.

Reply:
There's a weird bug that doesn't allow for string literals inside interpolated strings. It does work inside triple quoted strings.

This doesn't compile:
```fsharp
printfn $"It is raining: %s{if raining then "Yes" else "No"}"
``` 

This does:
```fsharp
printfn $"""It is raining: %s{if raining then "Yes" else "No"}"""`
```

https://sharplab.io/#v2:DYLgZgzgNALiBOBXAdlAJiA1AHwA7wEtkYxkACAEgCIYALAiMtAewFMJkByGMgd2fgBrMgG8CYMjCStJtVuSoBPdlTKtgEGVWTNVAXyoBYAFD4iJctSo16jFuz4DhYiVMQy68skpVqNWnX1rKiA=

Reply:
I'll change it to string interpolation with triple quotes.